### ISSUE TITLE
Test templates

### ISSUE THREAD

**Issue by LoveIsGrief** [balderdashy/sails-generate-model#4]
Hi,

It'd be great if this generator included test templates for (as described in the [documentation](http://sailsjs.org/#/documentation/concepts/Testing)) [mocha](http://mochajs.org/) tests e.g

`test/unit/model/Model.test.js`:

```javascript
Should = require("should")

describe('ModelModel', function() {

  describe('#create()', function() {
    it('should create a Model in the db', function (done) {
      Model.create({
             attritubeA: "A",
             attributeB: "B"
             // other attributes
         })
        .then(function(results) {
          Should.exist(results);
          // some tests
          done();
        })
        .catch(done);
    });
  });

});
```

**Comment by sailsbot** [comment]
Thanks for posting, @LoveIsGrief.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
